go.mod

```diff
@@ -306,7 +306,7 @@ replace (
 	// release/v1.15
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.20-0.20250815065500-a4fbafcae0dd
 	// develop
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.22.1-0.20251002045626-63f3de0a10b2
+	github.com/evmos/ethermint => github.com/thomas-nguy/ethermint v0.4.2-0.20251003145706-7847a6e9c1aa
```

Comment:
this will be updated once https://github.com/crypto-org-chain/ethermint/pull/722 get merged, right?